samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -7,7 +7,8 @@ add_q=Add Question:
 addr=Addresses:
 admin=Admin
 done=done
-ass_preview=Assessment Preview - This is an example student view of this assessment                                                            
+exit_preview=Exit Preview
+ass_preview=Assessment Preview - This preview provides an approximate view of the layout of the assessment for students. \The preview will <strong>not</strong> allow you to save or submit answers, display the timer if there is a time limit, or access any feedback. \The preview also does <strong>not</strong> take into account students' ability (or lack thereof) to view images or other media embedded in questions.
```

Comment:
I suggest we modify the last sentence to:

`The preview does <strong>not</strong> reflect whether students can access or view images or other media embedded in questions.`

Reply:
Done!

Reply:
Thanks for making those changes, @MRutea.